appliances/MinIO/appliance.sh

```diff
@@ -114,7 +121,7 @@ service_install()
 # Start MinIO-volumes
 # End MinIO-volumes
 EOF
-
+    edit_hosts
```

Comment:
maybe we do not need a function? or merge the logic of 
```

       # For each node, get IP and create entry using MinIO Multi-Node notation
        i=1
        for node in "${local_nodes[@]}"
        do
            local_ip=$(onegate vm show ${node} --json | jq -r '.VM.TEMPLATE.NIC[].IP')
            local_hosts+="${local_ip} ${local_subdomain}$((i))${local_domain}\n"
            i=$((i + 1))
        done
        msg info "Adding MinIO hostnames to /etc/hosts"
        sed -i "/# End MinIO hosts/i ${local_hosts}" /etc/hosts
```

into the function like setup_etc_hosts